doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -107,32 +290,41 @@ See also :func:`esmvalcore.preprocessor.seasonal_mean`.
 6. annual_mean
 --------------
 
-This function produces an annual or a decadal mean. The only argument is the
-decadal boolean switch. When this switch is set to true, this function
-will output the decadal averages.
+This function produces an annual or a decadal mean. It takes a single boolean
+switch as argument:
+* decadal: calculate decadal averages.
 
 See also :func:`esmvalcore.preprocessor.annual_mean`.
 
 
 7. regrid_time
 --------------
 
-This function aligns the time points of each component dataset so that the dataset
-iris cubes can be subtracted. The operation makes the datasets time points common and
-sets common calendars; it also resets the time bounds and auxiliary coordinates to
-reflect the artifically shifted time points. Current implementation for monthly
-and daily data; the frequency is set automatically from the variable CMOR table
-unless a custom frequency is set manually by the user in recipe.
+This function aligns the time points of each component dataset so that the
+dataset Iris cubes can be subtracted. The operation makes the datasets time
+points common and sets common calendars; it also resets the time bounds and
+auxiliary coordinates to reflect the artifically shifted time points. Current
+implementation for monthly and daily data; the frequency is set automatically
+from the variable CMOR table unless a custom frequency is set manually by the
+user in recipe.
 
+.. warning::
+   Link to function missing.
+
+
+Area operations
+===============
+
+.. warning::
+   Need to be adapted after renaming action in #1123
 
-Area manipulation
-=================
 The _area.py module contains the following preprocessor functions:
 
-* extract_region: Extract a region from a cube based on lat/lon corners.
-* zonal_means: Calculates the zonal or meridional means.
-* area_statistics: Calculates the average value over a region.
-* extract_named_regions: Extract a specific region from in the region cooordinate.
+1. extract_region: extract a region from a cube based on lat/lon corners.
+2. zonal_means: calculate the zonal or meridional means.
+3. area_statistics: calculate the average value over a region.
+4. extract_named_regions: Extract a specific region from in the region
+   cooordinate.
```

Comment:
That is not clear. What is the difference between "extract_region" and "extract_named_regions"?

Reply:
The first uses coordinate corners, the latter the region name. I fixed that